server/services/caService.js

```diff
@@ -21,6 +21,7 @@ module.exports = function createCaService(roService, lduActiveClient, { continue
     async getReasonForNotContinuing(bookingId, token) {
       if (!continueCaToRoFeatureFlag) {
         return null
+        // continueCaToRoFeatureFlag is defaulted to false in config so !continueCaToRoFeatureFlag means CA should proceed to RO without checking for errors
```

Comment:
"defaulted false in config" is only true in certain envs and isn't static. 
If you wanted to describe this it would be nicer to raise the layer of abstraction: "When this feature is disabled, we never block the CA from continuing the case"

Reply:
ok